easybuild/easyconfigs/v/VTK/VTK-8.2.0-EGL-CrayGNU-19.10-python3.eb

```diff
@@ -71,19 +66,25 @@ configopts += "-DCMAKE_CXX_COMPILER:PATH=`which CC` "
 #
 configopts += "-DVTK_USE_X=OFF -DVTK_OPENGL_HAS_EGL:BOOL=ON "
 configopts += "-DEGL_INCLUDE_DIR:PATH=/users/jfavre/Projects/EGL/code-samples/posts/egl_OpenGl_without_Xserver "
-configopts += "-DEGL_LIBRARY:FILEPATH=/opt/cray/nvidia/default/lib64/libEGL.so "
-configopts += "-DEGL_opengl_LIBRARY:FILEPATH=/opt/cray/nvidia/default/lib64/libOpenGL.so "
 
-modextrapaths = {'PYTHONPATH': ['lib/python%s/site-packages' % local_pyshortver]}
+if os.path.isfile('/opt/cray/nvidia/default/lib64/libEGL.so'):
+    configopts += "-DEGL_LIBRARY:FILEPATH=/opt/cray/nvidia/default/lib64/libEGL.so "
+    configopts += "-DEGL_opengl_LIBRARY:FILEPATH=/opt/cray/nvidia/default/lib64/libOpenGL.so "
+else:  # DOM os.environ['HOST'].startswith('dom'):
+    configopts += '-DEGL_LIBRARY:FILEPATH=/usr/lib64/libEGL.so '
+    configopts += '-DEGL_opengl_LIBRARY:FILEPATH=/usr/lib64/libOpenGL.so '
+
+modextrapaths = {'PYTHONPATH': ['lib/python%(pyshortver)s/site-packages']}
+
 # a long discussion with Victor, Matthias and Rafael lead to the following hard-wired variable.
 # should the cray-python module be fixed?
 modtclfooter = """
-prepend-path LD_LIBRARY_PATH "/opt/python/3.6.5.7/lib"
+prepend-path LD_LIBRARY_PATH "/opt/python/%(pyver)s/lib"
```

Comment:
> a long discussion with Victor, Matthias and Rafael lead to the following hard-wired variable

@jfavre is this comment still valid? Do you need to hard-wire the python version or is it fine to get it from `pyver`?

Reply:
should be absolutely fine to get it from pyver

Reply:
Thanks. So @Henrique I think it's better omit that comment from the new recipe.